QUICKSTART.md

```diff
@@ -12,41 +12,19 @@
 ## 1 Query data using Flight clients
 This process is the same if you launched the Dremio locally or via docker.
 
-### 1.1 Query your datasets with arrow flight client in python
-
-This lightweight Python client application connects to the Dremio Arrow Flight server endpoint. It requires the username and password for authentication. Developers can use admin or regular user credentials for authentication. Any datasets in Dremio that are accessible by the provided Dremio user can be queried. By default, the hostname is `localhost` and the port is `32010`. Developers can change these default settings by providing the hostname and port as arguments when running the client. Moreover, the tls option can be provided to establish an encrypted connection.
-
-> Note: Trusted certificates must be provided when the tls option is enabled.
+### 1.1 Query your datasets with Python
 > 
-#### 1.1.1 Prerequisites
-
--   Python 3
+#### 1.1.1 Getting Started
 
-#### 1.1.2 Instructions on using this Python sample application
+1. Install [Python 3](https://www.python.org/downloads/)
+2. Download and install the [dremio-flight-endpoint whl file](https://github.com/dremio-hub/arrow-flight-client-examples/releases)
+    - `python -m pip install <PATH TO WHEEL>` 
+3. Clone this repo and changed your working directory to: arrow-flight-client-examples/python/  
```

Comment:
We actually don't need this step, correct? They can just copy the example.py file to anywhere on their filesystem and run it, right? That's how dead simple we would like these instructions to be.

Reply:
Yeah, you're right. I was debating your method, but initially thought this method is more thorough. After thinking about it again, I think your way is simpler.